pytensor/_version.py

```diff
@@ -105,7 +102,7 @@ def run_command(commands, args, cwd=None, verbose=False, hide_stderr=False, env=
             return None, None
     else:
         if verbose:
-            print(f"unable to find command, tried {commands}")
+            print("unable to find command, tried %s" % (commands,))
```

Comment:
Why this?

Reply:
It was a versioneer script refresh, I've pinned the version in the pyproject.toml